Design1/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Minimal Blog Card</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="card">
+      <img src="./cactus_img.jpg"/>
+      <span class="tag">Design</span>
```

Comment:
try using css selectors like `.card span{....Your stylings here }` Here You have given separate class that is `tag`. class name is not correct if you are using this approach then name must be `card-tag and card-title`